packages/editor/src/client/blocks/embeds/polis.tsx

```diff
@@ -1,17 +1,19 @@
-import React from 'react'
+import React, {useEffect} from 'react'
+import {useScript} from '../../utility'
 
 export interface PolisEmbedProps {
   conversationID: string
 }
 
 export function PolisEmbed({conversationID}: PolisEmbedProps) {
+  const {load} = useScript(`https://pol.is/embed.js`, () => false, false)
+
+  useEffect(() => load(), [])
+
   return (
-    <iframe
-      src={`https://pol.is/${conversationID}`}
-      width="100%"
-      height="600"
-      scrolling="yes"
-      frameBorder="0"
-    />
+    <div
+      className="polis"
+      data-page_id={conversationID}
+      data-site_id="polis_site_id_ssFrYaCCx4UpbUX63X"></div>
```

Comment:
what is this id? should be static? and how does this attribute work?

Reply:
Good catch. I changed it